<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnComisionar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Comisionar</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pgBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblTarea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tarea</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>312</width>
      <height>183</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbIniB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio Blanco</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbIniA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio Negro</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFinB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Blanco</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFinA">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Negro</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAno">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="prefix">
     <string>AÃ±o: </string>
    </property>
    <property name="minimum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinMes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="prefix">
     <string>Mes: </string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>btnComisionar</zorder>
   <zorder>pgBar</zorder>
   <zorder>lblTarea</zorder>
   <zorder>calendario</zorder>
   <zorder>rbIniB</zorder>
   <zorder>rbIniA</zorder>
   <zorder>rbFinB</zorder>
   <zorder>rbFinA</zorder>
   <zorder>spinAno</zorder>
   <zorder>spinMes</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
